src/billing/Workspaces/Workspaces.tsx

```diff
@@ -0,0 +1,279 @@
+      backgroundColor: colors.light(0.3),
+      border: `solid 2px ${colors.danger(0.3)}`,
+      borderRadius: 5,
+    },
+  };
+
+  return (
+    <div style={expandedInfoStyles.row}>
+      <div style={expandedInfoStyles.title}>{title}</div>
+      <div style={expandedInfoStyles.details}>{details}</div>
+      {errorMessage && <div style={expandedInfoStyles.errorMessage}>{errorMessage}</div>}
+    </div>
+  );
+};
+
+interface WorkspaceCardProps {
+  workspace: WorkspaceInfo;
+  billingAccountDisplayName: string | undefined;
+  billingProject: BillingProject;
+  billingAccountStatus: false | BillingAccountStatus;
+  isExpanded: boolean;
+  onExpand: () => void;
+}
+
+const WorkspaceCard: React.FC<WorkspaceCardProps> = memoWithName('WorkspaceCard', (props: WorkspaceCardProps) => {
+  const { workspace, billingProject, billingAccountStatus, billingAccountDisplayName, isExpanded, onExpand } = props;
+  const { namespace, name, createdBy, lastModified, googleProject, errorMessage } = workspace;
+  const workspaceCardStyles = {
+    field: {
+      ...Style.noWrapEllipsis,
+      flex: 1,
+      height: '1.20rem',
+      width: `calc(50% - ${(workspaceLastModifiedWidth + workspaceExpandIconSize) / 2}px)`,
+      paddingRight: '1rem',
+    },
+    row: { display: 'flex', alignItems: 'center', width: '100%', padding: '1rem' },
+    expandedInfoContainer: { display: 'flex', flexDirection: 'column', width: '100%' } satisfies CSSProperties,
+  };
+  const billingDetailsId = useUniqueId('details');
+
+  return (
+    <div role='row' style={{ ...Style.cardList.longCardShadowless, padding: 0, flexDirection: 'column' }}>
+      <div style={workspaceCardStyles.row}>
+        {billingAccountStatus && (
+          <div role='cell'>
+            <Icon {...getBillingAccountIconProps(billingAccountStatus)} />
+          </div>
+        )}
+        <div
+          role='rowheader'
+          style={{
+            ...workspaceCardStyles.field,
+            display: 'flex',
+            alignItems: 'center',
+            paddingLeft: billingAccountStatus ? '1rem' : '2rem',
+          }}
+        >
+          <Link
+            style={Style.noWrapEllipsis}
+            href={Nav.getLink('workspace-dashboard', { namespace, name })}
+            onClick={() => {
+              Ajax().Metrics.captureEvent(Events.billingProjectGoToWorkspace, {
+                workspaceName: name,
+                ...extractBillingDetails(billingProject),
+              });
+            }}
+          >
+            {name}
+          </Link>
+        </div>
+        <div role='cell' style={workspaceCardStyles.field}>
+          {createdBy}
+        </div>
+        <div role='cell' style={{ height: '1rem', flex: `0 0 ${workspaceLastModifiedWidth}px` }}>
+          {Utils.makeStandardDate(lastModified)}
+        </div>
+        <div role='cell' style={{ flex: `0 0 ${workspaceExpandIconSize}px` }}>
+          {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
```

Comment:
When the file was converted to TSX, we started getting a linting error about Link not having an href. The error says you should use a button.

In the case of no href and an `onClick` callback being passed, we do add the button role to the `div` that is rendered. So there is not an actual accessibility error… this is just the linting guessing that there _will_ be one.

Some folks have gotten around this by using `Clickable`, but that impacts styling (both hover and regular styling). Others have suppressed the linting warning. I am going with suppression over duplicating styles, but I think the best longterm fix would be to create a new component for this usage so it is clear that we are rendering a button (with appropriate aria attributes) styled as a link.